cacti/tests/compose/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quantumobject/docker-cacti
+FROM quantumobject/docker-cacti:1.2.8
```

Comment:
Is that because it's not working for `latest`?

If so, we should probably fix for latest (maybe in another PR). It's helpful to test `latest` when possible.

Reply:
It is broken because of changes not in cacti but on the image itself. I dont think is a good idea to test on latest because you cannot control when it breaks. We should only bump base docker image when either:

a) We've checked that is safe to do so
b) We are doing active development on the integration

Otherwise, with more than a hundred integrations is impossible to keep a green master. If master is never green then actual breakages are going to get buried and ignored